### PR TITLE
Add more metrics

### PR DESCRIPTION
**Pull request by pvalsecc**

### CODE REVIEW

File: tilecloud_chain/timedtilestore.py [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+    def _get_stats_name(self, func_name, tile=None):
+        if tile and 'layer' in tile.metadata:
+            return [self._stats_name, tile.metadata['layer'], func_name]
+        else:
+            return [self._stats_name, func_name]
+
+    def _time_iteration(self, generator, func_name):
+        while True:
+            try:
+                timer = stats.timer()
```
Comment (sbrunner) [inline tilecloud_chain/timedtilestore.py]: Why not using the `_time_iteration...`?

Comment (pvalsecc) [reply tilecloud_chain/timedtilestore.py]: In `_time_iteration` itself?

Comment (pvalsecc) [reply tilecloud_chain/timedtilestore.py]: Because I don't know the name of the metric in advance.

**Comment by pvalsecc** [comment]
OK, @sbrunner, I've fixed all your comments.

**Pull request merged**